instrumentation/jetty-httpclient/jetty-httpclient-9.2/javaagent/jetty-httpclient-9.2-javaagent.gradle

```diff
@@ -0,0 +1,26 @@
+apply plugin: "otel.javaagent-instrumentation"
```

Comment:
```suggestion
plugins {
  id("otel.javaagent-instrumentation")
}
```

Reply:
Ah missed it there yeah if you can use that pattern would be great

Reply:
using `plugins {}` for all gradle files now